packages/calcite-components/src/components/action/action.tsx

```diff
@@ -265,21 +265,10 @@ export class Action
     const iconNode = calciteLoaderNode || calciteIconNode;
     const hasIconToDisplay = iconNode || el.children?.length;
 
-    const slotContainerNode = (
-      <div
-        class={{
-          [CSS.slotContainer]: true,
-          [CSS.slotContainerHidden]: loading,
```

Comment:
See lines 103 - 107 of action.scss

Reply:
Hey @alisonailea can we revert this incase this refactor might break something? The change isn't the same because it's hiding the slotted element and not the internal div. The slotted element is in light dom so it could have some css conflict. It's also more feeble with the css if we add another slot then it would hide it when loading.